conf/apigen/templates/@layout.latte

```diff
@@ -162,20 +159,20 @@ the file LICENSE.md that was distributed with this source code.
 	<div id="left">
 
 	<div id="versions" class="dropdown">
-		<a class="dropdown-toggle" data-toggle="dropdown" data-target="#" href="/master">
+		<a class="dropdown-toggle" data-toggle="dropdown" data-target="#" href="/en/master">
 			Version: <span class="version">master</span>
 			<b class="caret"></b>
 		</a>
 		<ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
-			<li><a tabindex="-1" href="/master">master</a></li>
-			<li><a tabindex="-1" href="/3.6">3.6</a></li>
-			<li><a tabindex="-1" href="/3.5">3.5</a></li>
-			<li><a tabindex="-1" href="/3.4">3.4</a></li>
-			<li><a tabindex="-1" href="/3.3">3.3</a></li>
-			<li><a tabindex="-1" href="/3.2">3.2</a></li>
-			<li><a tabindex="-1" href="/3.1">3.1</a></li>
-			<li><a tabindex="-1" href="/3.0">3.0</a></li>
-			<li><a tabindex="-1" href="/2.4">2.4</a></li>
+			<li><a tabindex="-1" href="/en/master">master</a></li>
+			<li><a tabindex="-1" href="/en/3.6">3.6</a></li>
+			<li><a tabindex="-1" href="/en/3.5">3.5</a></li>
+			<li><a tabindex="-1" href="/en/3.4">3.4</a></li>
+			<li><a tabindex="-1" href="/en/3.3">3.3</a></li>
+			<li><a tabindex="-1" href="/en/3.2">3.2</a></li>
+			<li><a tabindex="-1" href="/en/3.1">3.1</a></li>
+			<li><a tabindex="-1" href="/en/3.0">3.0</a></li>
+			<li><a tabindex="-1" href="/en/2.4">2.4</a></li>
```

Comment:
This isn't highlighting the current version if you click around a couple of pages - there's a JS event handler that looks at this in `js/combined.js` line 1282 which will need the regex updated to handle the en prefix